src/main/java/uk/gov/companieshouse/web/pps/controller/pps/ConfirmationController.java

```diff
@@ -20,11 +22,8 @@
 import uk.gov.companieshouse.web.pps.service.penaltypayment.PayablePenaltyService;
 import uk.gov.companieshouse.web.pps.session.SessionService;
 
-import jakarta.servlet.http.HttpServletRequest;
-import java.util.Map;
-
 @Controller
-@RequestMapping("/late-filing-penalty/company/{companyNumber}/penalty/{penaltyId}/confirmation")
+@RequestMapping("/late-filing-penalty/company/{companyNumber}/penalty/{penaltyId}/payment/{paymentReference}/confirmation")
```

Comment:
I think this new path parameter should be renamed to be `/payable/{payableRef}` as looking at the `payable_resources` mongo collection and the penalty-payment-api we have this term already.

Note: This can be seen in other places in the penalty-payment-web code as penaltyId i.e. `payable/{penaltyId}`
https://github.com/companieshouse/penalty-payment-web/blob/master/src/main/java/uk/gov/companieshouse/web/pps/service/penaltypayment/impl/PayablePenaltyServiceImpl.java#L24
So I think it should also be renamed in: PayablePenaltyServiceImpl / PayablePenaltyServiceImplTest / PPSTestUtility.

https://github.com/companieshouse/penalty-payment-api/blob/master/README.md
| **GET**   | `/company/{company_number}/penalties/late-filing/payable/{id}`         | Get a payable resource                                                |